sample/peer/cmd/root.go

```diff
@@ -67,6 +67,14 @@ func init() {
 	rootCmd.PersistentFlags().String("keys", defKeysFile, "keyset file")
 	must(viper.BindPFlag("keys",
 		rootCmd.PersistentFlags().Lookup("keys")))
+
+	rootCmd.PersistentFlags().String("logging-level", "", "logging level")
+	must(viper.BindPFlag("logging.level",
+		rootCmd.PersistentFlags().Lookup("logging-level")))
+
+	rootCmd.PersistentFlags().String("logging-file", "", "logging file")
+	must(viper.BindPFlag("logging.file",
+		rootCmd.PersistentFlags().Lookup("logging-file")))
```

Comment:
Adding this code to `root.go` would suggest that `peer request` command also supports those flags, whereas it doesn't. Please consider moving this code to `run.go`. Later on, we can add proper logging to the client module as well and support those flags for both `run` and `request` commands.

Reply:
Right, I'll move this to run.go.